### PR TITLE
add vgp training skeleton from workflow (hidden)

### PR DESCRIPTION
**Pull request by astrovsky01**
Add hidden training for vgp assembly skeleton

### CODE REVIEW

**Review by hexylena — DISMISSED** [review]

File: topics/assembly/tutorials/vgp_genome_assembly/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,2263 @@
...
+{: .details}
+
+A big step can have several subsections or sub steps:
+
+
+## Sub-step with **Parse parameter value**
+
+> ### {% icon hands_on %} Hands-on: Task description
+>
+> 1. {% tool [Parse parameter value](param_value_from_file) %} with the following parameters:
```
Comment (hexylena) [inline topics/assembly/tutorials/vgp_genome_assembly/tutorial.md:139]: Can you double check these @bgruening

Comment (bgruening) [reply topics/assembly/tutorials/vgp_genome_assembly/tutorial.md:139]: afaik those are fine. Those are Galaxy internal, the new expression tools.

Comment (hexylena) [reply topics/assembly/tutorials/vgp_genome_assembly/tutorial.md:139]: ok, the  I'll add to the allowlist.

**Review by hexylena — APPROVED** [review]

**Comment by bgruening** [comment]
Danke!

**Pull request merged**